internal/pipeline/executor.go

```diff
@@ -309,8 +308,8 @@ func filterMetaData(resources []*yaml.RNode) []*yaml.RNode {
 
 // fnFilters returns chain of functions that are applicable
 // to a given pipeline.
-func fnFilters(p *Pipeline) ([]kio.Filter, error) {
-	filters, err := p.fnChain()
+func fnFilters(p *v1alpha2.Pipeline, pkgPath string) ([]kio.Filter, error) {
```

Comment:
As a convention I am thinking, if we want to use short name for `Pipeline`, we should use `pl` instead of `p`. We can reserve `p` for pkg.